tt_metal/hw/inc/blackhole/dev_mem_map.h

```diff
@@ -25,75 +25,72 @@
 
 /////////////
 // RISC-V Address map definition (hardware)
-#define MEM_L1_BASE           0x0
-#define MEM_L1_SIZE           (1464 * 1024)
+#define MEM_L1_BASE 0x0
+#define MEM_L1_SIZE (1464 * 1024)
 
-#define MEM_ETH_BASE          0x0
-#define MEM_ETH_SIZE          (256 * 1024)
+#define MEM_ETH_BASE 0x0
+#define MEM_ETH_SIZE (256 * 1024)
 
-#define MEM_LOCAL_BASE        0xFFB00000
-#define MEM_BRISC_LOCAL_SIZE  (4 * 1024)
+#define MEM_LOCAL_BASE 0xFFB00000
+#define MEM_BRISC_LOCAL_SIZE (4 * 1024)
 #define MEM_NCRISC_LOCAL_SIZE (4 * 1024)
 #define MEM_IERISC_LOCAL_SIZE (4 * 1024)
-#define MEM_TRISC_LOCAL_SIZE  (2 * 1024)
+#define MEM_TRISC_LOCAL_SIZE (2 * 1024)
 
-#define MEM_NCRISC_IRAM_BASE  0xFFC00000
-#define MEM_NCRISC_IRAM_SIZE  (16 * 1024)
+#define MEM_NCRISC_SIZE (16 * 1024)
```

Comment:
is this used?  Or just MEM_NCRISC_FIRMWARE_SIZE below?

Reply:
ah redundant, I am using this in tensix-memory.ld but I can use `MEM_NCRISC_FIRMWARE_SIZE`